### ISSUE TITLE
How do I make this interpret mon, tue, thu etc as the most recent day instead of the nearest day?

### ISSUE THREAD

**Issue by kamal2311** [wanasit/chrono#230]
How do I make chrono-node interpret mon, tue, thu etc as the most recent day( in the past) instead of the nearest day which is sometimes in future?

**Comment by amuelli** [comment]
As described in the [README](https://github.com/wanasit/chrono#reference-date):
> Chrono lets you define a reference date using `chrono.parse(text, ref)` and `chrono.parseDate(text, ref)`.

For your case just use the current date a week ago as reference date:
```js
var moment = require('moment');

chrono.parse('mon', moment().subtract(1, 'week'); 
```

**Comment by kamal2311** [comment]
This 1 week back ref date solution would disturb the interpretation of terms like 'yesterday' ' 2 days ago' etc.

**Comment by amuelli** [comment]
You are right it effects other parsers. There is also still the issue that e.g.`tue` by default will be interpreted as the closest Tuesday to the ref date, so that with the 1 week back ref solution it might even be 2 weeks back.
I think you have to write your own refiner to accomplish exactly what you need, something like this:
https://github.com/wanasit/chrono/issues/19#issuecomment-164419896
